golden/clr-angular.d.ts

```diff
@@ -219,24 +219,31 @@ export declare class ClrControlHelper {
 export declare class ClrDatagrid<T = any> implements AfterContentInit, AfterViewInit, OnDestroy {
     SELECTION_TYPE: typeof SelectionType;
     allSelected: boolean;
+    calculationRows: ViewContainerRef;
```

Comment:
Could you go over the new properties and methods in the golden file, and check for each one if it should be private, or if it needs to be public but underscore-prefixed?